Core/CrossCuttingConcerns/Logging/Log4Net/LoggerServiceBase.cs

```diff
@@ -0,0 +1,53 @@
+﻿using log4net;
+using log4net.Repository;
+using System.Reflection;
+using System.Xml;
+
+namespace WebNetSample.Core.CrossCuttingConcerns.Logging.Log4Net;
+
+public class LoggerServiceBase
+{
+    private ILog _log;
+
+    public LoggerServiceBase(string name)
+    {
+        var xmlDocument = new XmlDocument();
+        xmlDocument.Load(File.OpenRead(
+                Path.Combine(
```

Comment:
I think that you can easily move `Path.Combine(...)` to separate variable and put it here to avoid much code nesting. It will be also easier to debug